### PR TITLE
type: "script" mode

### PR DESCRIPTION
**Pull request by guybedford**
In jspm, `"type": "script"` is now taken to mean that the JS files should be left entirely untouched and unprocessed by jspm. As in, they should not be optimized as modules.

SystemJS exactly fits this definition as SystemJS itself is shipped as a script goal.

There is no other script or spec for `"type": "script"`, but there doesn't need to be - it is a jspm specific feature. In Node.js files will be loaded as CommonJS as usual.

### CODE REVIEW

File: test/package.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "type": "module"
```
Comment (joeldenning) [inline test/package.json:2]: What is this file for? Is it used in tests? Looks like it might be able to be removed.

**Review by joeldenning — APPROVED** [review]

File: test/package.json (hunk 1 above)
Comment (guybedford) [reply test/package.json:2]: This makes a package scope in the `test` folder where all these files are treated as modules again. I just had to add one to the `src` folder as well similarly to get the tests to pass.

Comment (guybedford) [reply test/package.json:2]: (This is for Node.js execution)

Comment (joeldenning) [reply test/package.json:2]: 👍

**Pull request merged**